specification/providerhub/resource-manager/Microsoft.ProviderHub/preview/2021-06-01-preview/providerhub.json

```diff
@@ -2431,6 +2435,10 @@
           "description": "Indicates whether expedited rollout is enabled/disabled",
           "type": "boolean"
         }
+      },
+      "systemData": {
+        "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData",
+        "readOnly": true
       }
```

Comment:
nit - remove. This is just a property def.
"DefaultRollout": { } is the actual resource def and is updated.